.cargo/config

```diff
@@ -1,3 +1,7 @@
-[target.x86_64-apple-darwin]
-linker = "x86_64-apple-darwin15-clang"
-ar = "x86_64-apple-darwin15-ar"
\ No newline at end of file
+# HACK: Disabled to allow for building on Darwin
```

Comment:
Unsure how we want to handle this.

Change was because the current setup breaks compilation for me on Darwin for Darwin target. I can override using ENV vars and get it working or comment out 🤔.

My recommendation is to provide people with the right ENV vars they can set depending on their build platform to `DO THE RIGHT THING`. Then we remove this file.

Sound good?

Reply:
I could cherry pick those changes over from Batch #2.

Reply:
Perhaps this is best solved by either `darwin.env` and `linux.env` files you can source and then your shell is setup to work right. Or, a shell script in `bin/setenv` that outputs the right env depending on which build system you run :D.

TODO: then update docs to indicate dev environment setup.

Reply:
See ./bin/release and ./bin/setenv and docs/index.md under "Cross Compilation" for info 👯 

DONE